### PR TITLE
Add 'geoserver_init' service

### PR DESCRIPTION
**Pull request by chrismayer**
This adds a `geoserver_init` service in order to initialize the SAUBER GeoServer:

  - Adapting the login credentials
  - Creating the workspaces
  - Creating the PostGIS DataStore connect
  - Creating the stations layer

The service is executed directly after the `geoserver` service is fully available (ensured by a `wait.for.sh` script).

### CODE REVIEW

File: geoserver_init/js-utils/logging.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Logs message embedded in a big frame.
```
Comment (JakobMiksch) [inline geoserver_init/js-utils/logging.js:2]:
```suggestion
 * Log message embedded in a big frame.
```

File: geoserver_init/js-utils/logging.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ */
+export function framedBigLogging(msg) {
+  console.log('##############################################################');
+  console.log(msg);
+  console.log('##############################################################');
+  console.log();
+}
+
+/**
+ * Logs message embedded in a medium frame.
```
Comment (JakobMiksch) [inline geoserver_init/js-utils/logging.js:14]:
```suggestion
 * Log message embedded in a medium frame.
```

File: geoserver_init/index.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+ * @param {Function} callback
+ */
+async function asyncForEach(array, callback) {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
+  }
+}
+
+/**
+ * Exits scrpt and logs an error message.
```
Comment (JakobMiksch) [inline geoserver_init/index.js:155]:
```suggestion
 * Exits script and logs an error message.
```

**Review by JakobMiksch — APPROVED** [review]
I  cannot say much about the Docker stuff, but the JS part looks good to me. Only a few typos.

File: geoserver_init/index.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    console.info('Successfully created user', user);
+  }
+
+  const roleAssigend = await grc.security.associateUserRole(user, role);
+  if (roleAssigend) {
+    console.info(`Successfully added role ${role} to user ${user}`);
+  }
+
+  // disable user
+  const adminDisabled = await grc.security.updateUser('admin', 'geoserver', false);
```
Comment (JakobMiksch) [inline geoserver_init/index.js:84]: `admin` and `geoserver` should be replaced by the respective variables that are declared at the beginning of the document

**Comment by chrismayer** [comment]
Thanks for the review @JakobMiksch! Addressed your comments in 718b7ad.

@jdk24 is this good to go for you as well?

**Review by jdk24 — APPROVED** [review]
All workspaces, layers and datastores created successfully. Please merge at will!

**Comment by chrismayer** [comment]
Thanks for your reviews, @JakobMiksch & @jdk24 I'm going to merge now.

**Pull request merged**